.gitignore

```diff
@@ -3,10 +3,8 @@ bootrom
 target
 *.jar
 *.stamp
-/vsim
-/verisim/generated-src*
-/verisim/simulator-*
-/verisim/verilator
-simv*
```

Comment:
I don't understand removing this based on your comment. Are you saying your personal gitignore ignores `vsim` and `verisim` or are you saying you don't look at all the spew this will cause in `git status`

Reply:
Sorry this is unclear from my comment. Here I removed the files from the `.gitignore` because in the `sims/verisim/.gitignore` and `sims/vsim/.gitignore`, I use `*` to by default ignore all files in the `sims/<verisim/vsim>/`. Then in those files I selectively "unignore" all relevant files". I think this is better so that the `.gitignore`s in the simulation directories tell git what to ignore rather than the top-level `.gitignore`

Reply:
Ah makes sense on my last skim I didn't see the changes to the sim directory gitignores.